specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/examples/ContactProfileCreate.json

```diff
@@ -0,0 +1,238 @@
+            "subnetId": "/subscriptions/c1be1141-a7c9-4aac-9608-3c2e2f1152c3/resourceGroups/contoso-Rgp/providers/Microsoft.Network/virtualNetworks/contoso-vnet/subnets/orbital-delegated-subnet"
+          },
+          "thirdPartyConfigurations": [
+            {
+              "providerName": "KSAT",
+              "missionConfiguration": "Ksat_MissionConfiguration"
+            },
+            {
+              "providerName": "VIASAT",
+              "missionConfiguration": "Viasat_Configuration"
+            }
+          ],
+          "links": [
+            {
+              "name": "contoso-uplink",
+              "polarization": "LHCP",
+              "direction": "uplink",
+              "gainOverTemperature": 0,
+              "eirpdBW": 45,
+              "channels": [
+                {
+                  "name": "contoso-uplink-channel",
+                  "centerFrequencyMHz": 2250.0,
+                  "bandwidthMHz": 2.0,
+                  "endPoint": {
+                    "ipAddress": "10.1.0.4",
+                    "endPointName": "ContosoTest_Uplink",
+                    "port": "50000",
+                    "protocol": "TCP"
+                  },
+                  "modulationConfiguration": null,
+                  "demodulationConfiguration": null,
+                  "encodingConfiguration": null,
+                  "decodingConfiguration": null
+                }
+              ]
+            },
+            {
+              "name": "contoso-downlink",
+              "polarization": "RHCP",
+              "direction": "downlink",
+              "gainOverTemperature": 25.0,
+              "eirpdBW": 0.0,
+              "channels": [
+                {
+                  "name": "contoso-downlink-channel",
+                  "centerFrequencyMHz": 8160.0,
+                  "bandwidthMHz": 15.0,
+                  "endPoint": {
+                    "ipAddress": "10.1.0.5",
+                    "endPointName": "ContosoTest_Downlink",
+                    "port": "50001",
+                    "protocol": "UDP"
+                  },
+                  "modulationConfiguration": null,
+                  "demodulationConfiguration": null,
+                  "encodingConfiguration": null,
+                  "decodingConfiguration": null
+                }
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "201": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/Microsoft.Orbital/operationResults/4e2ffff7-b331-4fcb-ab11-b5fa49368188?api-version=2022-11-01"
+      },
+      "body": {
+        "location": "eastus2",
+        "tags": null,
+        "id": "/subscriptions/c1be1141-a7c9-4aac-9608-3c2e2f1152c3/resourceGroups/contoso-Rgp/providers/Microsoft.Orbital/contactProfiles/CONTOSO-CP",
+        "name": "CONTOSO-CP",
+        "type": "Microsoft.Orbital/contactProfiles",
+        "properties": {
+          "provisioningState": "created",
```

Comment:
@MicrosoftJim this should be succeeded right?

Reply:
Proper casing here is PascalCase Succeeded though

Reply:
we had an enum case mismatch issue with one of the pipeline tests and that's why we decided to change the casing